src/app/components/modules/Header.jsx

```diff
@@ -210,9 +210,7 @@ class Header extends React.Component {
                                     </Link>
                                 </li>
                                 <li className="Header__top-steemit show-for-medium noPrint"><Link to={logo_link}><span className="beta fade-in--10">beta</span></Link></li>
-                                
-                    
-                                {(topic_link || user_name || page_name) && sort_order }
+                                {(topic_link || user_name || page_name) && sort_order && <li className="delim show-for-small-only"></li>}
```

Comment:
what exactly is this fixing?

Reply:
Right now, there is a new bug introduced where if you click on a tag on desktop, this appears. Reverting this back to the original fixes it.

![screen shot 2017-11-13 at 1 56 54 pm](https://user-images.githubusercontent.com/1121139/32743537-b0dc8174-c87a-11e7-9f7a-41497cf4ebed.png)

Reply:
looks like it used to have content though; a pipe `|` https://github.com/steemit/condenser/commit/a528595192e76c19fc670735f7c783fbabca03b3